<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NacelleGeometryDlg</class>
 <widget class="QDialog" name="NacelleGeometryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nacelle geometry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Nacelle inlet lip</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="cbGenerateLip">
        <property name="text">
         <string>Generate inlet lip</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Axial offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="sbAxialOffset">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.300000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Radial offset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sbRadialOffset">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>0.900000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.300000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Shape coefficient</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="sbShapeCoef">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.200000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbApply">
     <property name="text">
      <string>&amp;Apply</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbGenerateLip</tabstop>
  <tabstop>sbAxialOffset</tabstop>
  <tabstop>sbRadialOffset</tabstop>
  <tabstop>sbShapeCoef</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbGenerateLip</sender>
   <signal>clicked(bool)</signal>
   <receiver>sbAxialOffset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGenerateLip</sender>
   <signal>clicked(bool)</signal>
   <receiver>sbRadialOffset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGenerateLip</sender>
   <signal>clicked(bool)</signal>
   <receiver>sbShapeCoef</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked(bool)</signal>
   <receiver>NacelleGeometryDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
